src/main/java/uk/gov/ons/ctp/response/action/export/service/NotificationAndManifestFileCreator.java

```diff
@@ -51,34 +55,67 @@ public void uploadData(
       ExportJob exportJob,
       String[] responseRequiredList,
       int actionCount) {
+
     if (actionCount == 0) {
       return;
     }
 
-    final String now = FILENAME_DATE_FORMAT.format(clock.millis());
-    String filename = String.format("%s_%s.csv", filenamePrefix, now);
+    String now = FILENAME_DATE_FORMAT.format(clock.millis());
+    String csvFilename = String.format("%s_%s.csv", filenamePrefix, now);
+    String manifestFilename = csvFilename.replace(".csv", ".manifest");
```

Comment:
really pernickety, but I don't like the use of `.replace` here - if we change the csv filename to a different extension, this line breaks. My preference would be to have the filename built the same way as the other one... but maybe that's just a style thing.

Reply:
fixed